tests/MongoDB.Driver.Tests/RetryableWritesTests.cs

```diff
@@ -43,6 +43,26 @@ public void Insert_with_RetryWrites_true_should_work_whether_retryable_writes_ar
             }
         }
 
+        [SkippableFact]
+        public void Retryable_write_operation_should_throw_custom_exception_on_servers_using_mmapv1()
+        {
+            RequireSupportForRetryableWrites();
+            RequireServer.Check().StorageEngine("mmapv1");
+
+            using (var client = GetClient(builder =>  { }))
```

Comment:
Double spaces.
Also, isn't this method https://github.com/vincentkam/mongo-csharp-driver/pull/47/files#diff-ae6ac4c5e78bcf3e82fc0b6298622f8bR199 suitable here?

Reply:
Good point! Fixed